tests/Unit/Routes/StoreRoutesTest.php

```diff
@@ -390,10 +390,13 @@ public function testSessionUpdateGate()
             ->assertResponseStatus(200);
     }
 
-
     /** @test */
     public function testRegistrationFamilyUpdateGate()
     {
+        // TODO complete or discard
```

Comment:
Assuming you want this to go in as a todo

Reply:
yeah, it's on a list for tests-to-complete. :-(